tests/__snapshots__/storyshots.test.ts.snap

```diff
@@ -0,0 +1,2006 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
```

Comment:
should this snapshot file still be here?

Reply:
We could remove but it's not hurting anything really